### ISSUE TITLE
com.vaadin.flow.component cannot be resolved

### ISSUE THREAD

**Issue by kabruhns** [spring-guides/gs-crud-with-vaadin#19]
In MainView.java I am getting the following error when trying to extend VerticalLayout:

The hierarchy of the type MainView is inconsistent.
The type com.vaadin.flow.component.Component cannot be resolved. It is indirectly referenced from required .class files

**Comment by mstahv** [comment]
Did you add both vaadin-spring-boot-starter and the bom to your pom.xml? If so, then Vaadin dependencies are for some reason not loaded properly. Did your IDE pick up the changes? IntelliJ at least by default requires you to reload the configuration, whenever you add/remove dependencies.

**Comment by kabruhns** [comment]
Thank you for your fast answer!

I just cloned this repository to try it out. So the pom is the exact same.
I am using eclipse and I have already updated the maven project. I also checked "Force Update of Snapshots/Releases".

Also, I get errors in the CustomerEditor when importing Key and KeyNotifier (`The import com.vaadin.flow.component.KeyNotifier cannot be resolved`).

Seems like the entire `om.vaadin.flow.component` cannot be resolved.

Ever seen something similar?
